lib/livebook_web/live/home_live.ex

```diff
@@ -136,6 +140,17 @@ defmodule LivebookWeb.HomeLive do
           import_opts={@import_opts} />
       </.modal>
     <% end %>
+
+    <%= if @live_action == :edit_sessions do %>
```

Comment:
Interactions that do not require the server should be done on the client as most possible. For this one, I would do the following:

1. Generate the Action button and Checkboxes on the page
2. Put a class on them called .bulk-actions and hide them by default
3. When you click the "Edit" button, use [JS.remove_class](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.JS.html#remove_class/3) to remove the hiding class
4. On cancel, use JS.add_class to add it back

WDYT?

Reply:
@josevalim we need to keep track of the selected sessions though, no?

(and this particular line renders a modal when the route changes, so we need server for that)

Reply:
I'm confused here! I think this applies only to the menu toggle, right?